src/node_bindings.cc

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2016-2017 JSTP project authors. Use of this source code is
+// governed by the MIT license that can be found in the LICENSE file.
+
+#include <node.h>
+#include <v8.h>
+
+#include "common.h"
+#include "jsrs_parser.h"
+#include "jsrs_serializer.h"
+#include "packet_parser.h"
+
+using v8::Array;
+using v8::FunctionCallbackInfo;
+using v8::HandleScope;
+using v8::Isolate;
+using v8::Local;
+using v8::Object;
+using v8::String;
+using v8::Value;
+
+namespace jstp {
+
+namespace bindings {
+
+void Stringify(const FunctionCallbackInfo<Value>& args) {
+  Isolate* isolate = args.GetIsolate();
+
+  if (args.Length() != 1) {
+    THROW_EXCEPTION(TypeError, "Wrong number of arguments");
+    return;
+  }
+
+  HandleScope scope(isolate);
+
+  auto result = jstp::serializer::Stringify(isolate, args[0]);
+  args.GetReturnValue().Set(result);
+}
+
+void Parse(const FunctionCallbackInfo<Value>& args) {
+  Isolate* isolate = args.GetIsolate();
+
+  if (args.Length() != 1) {
+    THROW_EXCEPTION(TypeError, "Wrong number of arguments");
+    return;
+  }
+  if (!args[0]->IsString() && !args[0]->IsUint8Array()) {
+    THROW_EXCEPTION(TypeError, "Wrong argument type");
+    return;
+  }
+
+  HandleScope scope(isolate);
+
+  String::Utf8Value str(args[0]->ToString());
+  auto result = jstp::parser::Parse(isolate, str);
+  args.GetReturnValue().Set(result);
+}
+
+void ParseNetworkPackets(const FunctionCallbackInfo<Value>& args) {
+  Isolate* isolate = args.GetIsolate();
+
+  if (args.Length() != 2) {
+    THROW_EXCEPTION(TypeError, "Wrong number of arguments");
+    return;
+  }
+  if (!args[0]->IsString() || !args[1]->IsArray()) {
+    THROW_EXCEPTION(TypeError, "Wrong argument type");
+    return;
+  }
+
+  HandleScope scope(isolate);
+
+  String::Utf8Value str(args[0]->ToString());
+  auto array = Local<Array>::Cast(args[1]);
```

Comment:
I guess it is better to use `v8::Local::As` here to keep cast style consistent, but this one is optional and completely up to you.